index.html

```diff
@@ -183,36 +235,50 @@ <h1 class="project-title">Tonic</h1>
     <div class="card">
       <img src="./images/image-4.png" alt="" class="image">
       <div class="left-card1-block">
-        <h1 class="project-title">Multi-Post <br> Stories</h1>
+        <h1 class="project-title mobile">Multi-Post <br> Stories</h1>
+        <h1 class="project-title desktop">Uber <br> Navigation</h1>
```

Comment:
- Kindly, your card image is not given the padding as the Figma, please stick to the Figma as match as possible.👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206553615-dcfd1832-0e6f-42e6-bc23-f52f5d4e6dd1.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206553624-5fdad545-f35e-4baf-9254-4750af7a1bcb.png" width="200">|


😎️